bin/gruf

```diff
@@ -17,13 +17,20 @@
 #
 require 'rubygems'
 require 'bundler/setup'
-require 'gruf'
 require 'rails' rescue nil
 load 'config/environment.rb' if defined?(Rails)
+require 'gruf'
 
 begin
-  server = Gruf::Server.new
+  server = Gruf::Server.new(Gruf.server_options)
+  Gruf.services.each { |s| server.add_service(s) }
+  Gruf.interceptors.each { |i, opts| server.add_interceptor(i, opts) }
   server.start!
 rescue => e
-  Gruf.logger.fatal "FATAL ERROR: #{e.message} #{e.backtrace.join("\n")}"
+  msg = "FATAL ERROR: #{e.message} #{e.backtrace.join("\n")}"
+  if Gruf.logger
```

Comment:
Can Gruf.logger be a `Logger.new(STDOUT)` if one is not provided during configuration? That would allow us to omit this conditional here and simplify logging in general.

Reply:
Yup, it is. `self.logger = ::Logger.new(STDOUT)` in configuration.rb.

However, this conditional here catches it on the chance it fatals before that logger is setup.